crates/orderbook/openapi.yml

```diff
@@ -1077,6 +1093,12 @@ components:
             order slippage.
           type: integer
     SolverCompetitionResponse:
+      description: |
+        The settlements submitted by every solver for a specific auction.
+        The auction id corresponds to the id external solvers are provided
+        with. Auction ids are not globally unique, they are reused from time to
+        time. The backend keeps the competition information for a limited amount
+        of time.
```

Comment:
This part is no longer correct, right? (global uniqueness, stored for limited time)